index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>About Me | Marcus' Personal Webpage</title>
+  <link rel="stylesheet" type="text/css" href="../css/styles.css">
+</head>
+<body>
+
+  <div class="header">
+    <header>
+    <h1>Hi my name is Marcus</h1>
+    </header>
+  </div>
+
+  <article class="article-bio">
+    <h3>A short description of me!</h3>
+    <img src="/images/personal_site2.png" class="prof-pic">
+    <p>You think water moves fast? You should see ice. It moves like it has a mind. Like it knows it killed the world once and got a taste for murder. After the avalanche, it took us a week to climb out. Now, I don't know exactly when we turned on each other, but I know that seven of us survived the slide... and only five made it out. Now we took an oath, that I'm breaking now. We said we'd say it was the snow that killed the other two, but it wasn't. Nature is lethal but it doesn't hold a candle to man.</p>
+
+    <p>Look, just because I don't be givin' no man a foot massage don't make it right for Marsellus to throw Antwone into a glass motherfuckin' house, fuckin' up the way the nigger talks. Motherfucker do that shit to me, he better paralyze my ass, 'cause I'll kill the motherfucker, know what I'm sayin'? </p>
+  </article>
+
+  <article class="resources">
+    <h3>Resources I like</h3>
+    <div class="mdn">
+      <a href="https://developer.mozilla.org/en-US/"><img src="/images/mdn.png"></a>
+      <h5>Mozilla Developer Network</h5>
+      <p>This is the best online resource I have come across in all my year of programming.</p>
+    </div>
+    <div class="css">
+      <a href="https://css-tricks.com/"><img src="/images/CSS-Tricks.png"></a>
+      <h5>CSS TRICKS</h5>
+      <p>A great resource for whatever CSS problem you may encounter. Filled with excellent graphics as well.</p>
+    </div>
+
+
+
+  </article>
+
+  <article class="projects">
+    <h3>Future Projects</h3>
+
+    <section class="section1"></section>
```

Comment:
For both this and the section tag below it, they should wrap the project content.